internal/pkg/template/templates/workloads/services/backend/cf.yml

```diff
@@ -102,18 +138,31 @@ Resources:
               Action:
                 - "tag:GetResources"
               Resource: "*"
-      ManagedPolicyArns:
-        - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
-{{- end }}
+      {{- end}}
+  {{- end}}
+
   Service:
-    DependsOn:
-    - EnvControllerAction
     Metadata:
       'aws:copilot:description': 'An ECS service to run and maintain your tasks in the environment cluster'
     Type: AWS::ECS::Service
+    DependsOn:
+      - EnvControllerAction
+      {{- if and .ALBEnabled .HTTPSListener}}
+      - HTTPListenerRuleWithDomain
+      - HTTPSListenerRule
+      {{ else if .ALBEnabled}}
+      - HTTPListenerRule
+      {{- end}}
     Properties:
-{{include "service-base-properties" . | indent 6}}
+      {{- "\n"}}{{ include "service-base-properties" . | indent 6 }}
```

Comment:
Why do we need "\n" 🤔

Reply:
good question: [answer](https://stackoverflow.com/questions/46066524/how-can-i-place-go-template-actions-with-multi-line-output-at-any-indentation-le#answer-46066525)
so that i can indent it and have the template fold correctly in my editor 🙄

Reply:
can we use `nindent`? That seems nifty!

Reply:
we'd have to import https://github.com/Masterminds/sprig 😔

Reply:
😤